src/deluge/model/drum/drum.h

```diff
@@ -90,4 +90,6 @@ class Drum {
 	void getCombinedExpressionInputs(int16_t* combined);
 
 	virtual ModControllable* toModControllable() { return NULL; }
+
+	virtual void stopDelay() {}
```

Comment:
Why are you adding a function that's never over ridden and doesn't do anything except create a naming conflict on SoundDrum (e.g. is SoundDrum::stopDelay supposed to be Drum::stopDelay or Sound::stopDelay)?